### PR TITLE
Change first row label to 'wall', add 'cpu' row, update documentation.

### PR DESCRIPTION
**Pull request by snim2**
This PR resolves Issue #3.

### CODE REVIEW

File: format.c [hunk 1]
```diff
@@ -146,6 +146,13 @@ int cmp_rusage_stime(const void *x, const void *y)
 
 
 
+int cmp_double(const void *x, const void *y)
```
Comment (ltratt) [inline format.c]: I suspect we should call this cmp_cputime or similar, for consistency with the other cmp_\* functions?

File: format.c [hunk 2]
```diff
@@ -195,6 +205,8 @@ void format_like_time(Conf *conf)
       (long long) (user.tv_sec / conf->num_runs), (long long) ((user.tv_usec / 10000) / conf->num_runs));
 	fprintf(stderr, "sys  %9lld.%02lld\n",
       (long long) (sys.tv_sec / conf->num_runs), (long long) ((sys.tv_usec / 10000) / conf->num_runs));
+  fprintf(stderr, "cpu  %9lld.%02lld\n",
```
Comment (ltratt) [inline format.c]: Has the indentation gone wrong here?

File: format.c (hunk 1 above)
Comment (snim2) [inline format.c]: Yup, pushed a delta for this.

File: format.c (hunk 2 above)
Comment (snim2) [inline format.c]: This is a bit weird, I thought I had sublime set up to use 4-spaces as default. Anyhow, pushed a changset to fix this.

**Comment by ltratt** [comment]
Can we squash some of these commits down (doesn't have to be all of them, but we probably don't want indentation commits if we can avoid it) and do a force push to update the pull request?

**Comment by ltratt** [comment]
Oops, it looks like squashing has gone back beyond this branch?

**Comment by snim2** [comment]
Ugh git :-( 

That push was following the procedure here: http://blog.screensteps.com/recovering-from-a-disastrous-git-rebase-mistake

i.e. create a new branch that starts from just before the erroneous rebase, fix the rebase, delete old branch, rename new branch, force-push. So, I'm not sure what went wrong with that one...

**Comment by ltratt** [comment]
This has gone sufficiently wrong that github won't merge it automatically. We seem to have 17 commits in the pull request?

**Comment by snim2** [comment]
Well, this is extremely embarrassing. I'm not exactly sure what I did to screw up the PR, but the code on the `travis-ci` branch has already diverged from the commits here and will be hard to merge in once the refactoring there is complete.

Maybe it would be easier just to kill this branch altogether and change real->wall  and add cputime on the other branch. That would give a more linear history once the whole lot has been merged into master.

Ugh :(

**Comment by ltratt** [comment]
Yes, I suspect you're going to have to manually extract the correct patches. If you're lucky, you can create a new (temporary) branch, cherry pick out the good commits (4bb0155 and e02a1ba?), then push that temporary branch over this (thus making the pull request hunky dory).

**Comment by snim2** [comment]
Closing this PR; will deal with Issue #3 in another PR, now the testing work has diverged from this branch.

**Pull request closed without merging**